ui-options.html

```diff
@@ -0,0 +1,158 @@
+                  <h3>Get Learner / UI Options</h3>
+                  <p>
+                  For Wordpress: <a href="https://github.com/fluid-project/uio-wordpress-plugin">UI Options Wordpress Plugin</a>.
+                  </p>
+
+                  <p>
+                  For code projects: <a href="https://www.npmjs.com/package/infusion">using npm</a>, or <a href="https://github.com/fluid-project/infusion/releases">GitHub</a>.
+                  </p>
+                  <h3>Guides and Documentation</h3>
+                  <p>
+                  If installing Learner / UI Options using npm or from a github release, "<a href="https://docs.fluidproject.org/infusion/development/tutorial-userInterfaceOptions/UserInterfaceOptions.html">Setting Up User Interface Options</a>" will help installing and structuring your code (Note: this is not applicable if using the Wordpress plugin).
+                  </p>
+                  <p>
+                  Once Learner / UI Options is installed, <a href="https://docs.fluidproject.org/infusion/development/tutorial-userInterfaceOptions/WorkingWithUserInterfaceOptions.html">"Working with UI Options"</a> can help guide you through setting up your site's styles and markup to take advantage of Learner / UI Options.
+                  </p>
+                </article>
+
+                <!-- End UI Options Information Container -->
+
+            </div>
+
+            <footer class="floe-footer">
+                <div class="floe-content row">
+                    <div class="columns large-6 small-12">
+                        <div>
+                            <p>
+                                Floe is a project of the
+                                <a href="http://idrc.ocadu.ca">Inclusive Design Research Centre</a>, <a href="http://ocadu.ca">OCAD University</a>.<br/>
+                            </p>
+                            <p>
+                                <span class="floe-hewlett-logo">Funded by a grant from <a href="http://www.hewlett.org">The William and Flora Hewlett Foundation</a>.</span>
+                            </p>
+                        </div>
+                    </div>
+
+                    <div class="columns large-6 small-12">
+                        <address>
+                            205 Richmond St. W., Second Floor<br/>
+                            Toronto, Ontario, Canada<br/>
+                            M5V 1V3<br/>
+                            Telephone: (416) 977-6000 #3968<br/>
+                            Fax: (416) 977-9844
+                        </address>
+                        <p>
+                            <a href="mailto:jtreviranus@faculty.ocadu.ca">Jutta Treviranus</a> Principal Investigator, Director and Professor<br/>
+                            <a href="mailto:jmitchell@ocadu.ca">Jess Mitchell</a> Senior Manager, Research and Development<br/>
+                            <a href="mailto:cclark@ocadu.ca">Colin Clark</a> Lead Software Architect
```

Comment:
If you're updating all these pages anyways, maybe verify with @colinbdclark about the title used for him.

Reply:
I'm tempted in leaving this alone for now as there are a few other house cleaning items (such as updating the partner list on the front page).

Will issue a different PR for those fixes.